### PR TITLE
Feature/feature user login

### PR DESCRIPTION
**Pull request by santidrj**

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES-My-Pet-Care-Web/pullRequest?prid=5649863&bid=18499951)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # PES-My-Pet-Care-Web
 Software project from FIB (Facultat d'Informàtica de Barcelona)
+
+Link to web app: https://my-pet-care-web.herokuapp.com/
```
Comment (santidrj) [inline README.md:4]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES-My-Pet-Care-Web/pullRequest?prid=5649863)

**Review by AlbertPG — DISMISSED** [review]
Okey

**Review by xavier-campos-diaz — DISMISSED** [review]

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES-My-Pet-Care-Web/pullRequest?prid=5649863&bid=18499951)

File: README.md (hunk 1 above)
Comment (santidrj) [inline README.md:4]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES-My-Pet-Care-Web/pullRequest?prid=5649863)

**Review by xavier-campos-diaz — APPROVED** [review]

**Review by AlbertPG — APPROVED** [review]
Okey

**Pull request merged**